src/main/groovy/com/google/protobuf/gradle/ProtobufExtract.groovy

```diff
@@ -59,6 +64,8 @@ abstract class ProtobufExtract extends DefaultTask {
   private final ConfigurableFileCollection inputFiles = objectFactory.fileCollection()
   private final CopyActionFacade copyActionFacade = instantiateCopyActionFacade()
   private final ArchiveActionFacade archiveActionFacade = instantiateArchiveActionFacade()
+  private final ConfigurableFileCollection filteredProtoNonDirs = instantiateFilteredProtoNonDirs()
```

Comment:
nit: it can just be declared as `FileCollection`. But see my other comment for directory vs. non-directory.

Reply:
Moved to `FileCollection` to make it more obvious it is immutable.